tests/ui/borrowck/issue-81899.stderr

```diff
@@ -2,13 +2,8 @@ error[E0080]: evaluation of constant value failed
   --> $DIR/issue-81899.rs:11:5
    |
 LL |     panic!()
-   |     ^^^^^^^^ the evaluated program panicked at 'explicit panic', $DIR/issue-81899.rs:11:5
+   |     ^^^^^^^^ explicit panic
    |
-note: inside `f::<{closure@$DIR/issue-81899.rs:4:31: 4:34}>`
-  --> $DIR/issue-81899.rs:11:5
-   |
-LL |     panic!()
-   |     ^^^^^^^^
 note: inside `_CONST`
```

Comment:
This error does read strangely now, since the line above the note is not actually "inside `_CONST`"

Reply:
The note refers to the span below.

Reply:
Maybe "called from `_CONST`" would be better?

We probably want to always skip the first frame then, that should always coincide with the primary span.

Reply:
> Maybe "called from `_CONST`" would be better?

I'm ok with rewording the existing note message to make it clearer.

> We probably want to always skip the first frame then, that should always coincide with the primary span.

Looking at the logic in `report` there are cases where the span could be different, because it is passed in, but there doesn't seem to be any case where this triggers. I'd prefer to leave the code resilient to possible future changes, but I don't care one way or the other.

Reply:
Having pushed and seeing the number of `stderr` files affected, would you be ok with those wording changes being done in a follow up PR?

Reply:
This should probably be done only for const-eval, not for Miri. Miri already has its own logic here and the "inside" is correct there.

> Looking at the logic in report there are cases where the span could be different, because it is passed in, but there doesn't seem to be any case where this triggers. I'd prefer to leave the code resilient to possible future changes, but I don't care one way or the other.

The issue is that "called from" makes little sense for the topmost frame.